DX-1563 Add app manifest to example apps (#114)
miroapp/app-examples: README.md

  If you prefer, you can install and use equivalent alternatives, such as `yarn` or `pnpm`.
- For more information, visit our [developer documentation](https://developers.miro.com).

### App configuration

> All examples contain `app-manifest.yaml` file which can be used to quickly configure the example app.

Reviewer: ```suggestion
All examples include an [`app-manifest.yaml` file](https://developers.miro.com/docs/app-manifest) to quickly configure the example app.
```

> - Go to [App settings](https://miro.com/app/settings/user-profile/apps)
> - Click on Edit in manifest
> - Paste the yaml configuration and click Save

Reviewer: ```suggestion
- Go to [App settings](https://miro.com/app/settings/user-profile/apps).
- Click **Edit in manifest**.
- Paste the YAML configuration, and click **Save**.
```
